export const authErrors = {
  userAlreadyExists: 'User already exists',
};
